go/pkg/client/bytestream.go

```diff
@@ -26,33 +48,52 @@ func (c *Client) WriteBytes(ctx context.Context, name string, data []byte) error
 	return err
 }
 
+// WriteBytesWithOptions uploads a byte slice with a given resource name to the CAS
+// at an arbitrary offset but retries still resend from the specified offset.
+func (c *Client) WriteBytesWithOptions(ctx context.Context, name string, data []byte, opts ...ByteStreamWriteOption) (int64, error) {
```

Comment:
s/WriteBytesWithOptions/WriteBytesAtRemoteOffset? I think that makes it clearer what this is doing.

Reply:
Done.